docs/modules/deployment/pages/core/centos-rhel8/selinux.adoc

```diff
@@ -0,0 +1,70 @@
+
+If you have SELinux enabled on your {page-component-title} server, there must be a policy configured to allow {page-component-title} to bind to the ICMP service.
+
+To verify that you have SELinux running and enabled, use the `sestatus` command.
+If enabled, you should see the following:
+
+[source, console]
+----
+SELinux status:                 enabled
+SELinuxfs mount:                /sys/fs/selinux
+SELinux root directory:         /etc/selinux
+Loaded policy name:             targeted
+Current mode:                   enforcing
+Mode from config file:          enforcing
+Policy MLS status:              enabled
+Policy deny_unknown status:     allowed
+Memory protection checking:     actual (secure)
+Max kernel policy version:      33
+----
+
+Use `audit2allow -a` to verify if {page-component-title} violates SELinux policies.
+
+NOTE: If the `audit2allow` command is missing, run `dnf install policycoreutils-devel` to install package.
+
+Look for entries preventing bind ICMP sockets to bind:
```

Comment:
This line doesn't read very well; I'm not 100% sure what it's trying to say.

Reply:
@indigo423 , could you provide some feedback here?

Reply:
What it tries to say, the audit2allow command will give the user the output which SELinux policies prevent binding ICMP sockets. These policies need to be explicitly allowed from the user.